eclair-core/src/main/resources/reference.conf

```diff
@@ -49,6 +49,7 @@ eclair {
     payment_secret = optional
     basic_mpp = optional
     option_support_large_channel = optional
+    trampoline_payment = disabled
```

Comment:
Aren't we also missing `option_static_remotekey = disabled` and `keysend = disabled` if we need to specify all features?

Reply:
This shows the ambiguity that "not defined by default" == "defined as disabled". 🤷‍♂️

Reply:
I tested and it doesn't look like we need to explicitly disable not-defined-by-default features...
So we should either:

- remove that line and keep `option_static_remotekey` and `keysend` absent as well
- or include them all?

I slightly prefer the second option where the `reference.conf` lists all supported features, with some of them disabled.
It makes it easier for users to figure out what features they can enable/disable without having to look at the actual code.

Reply:
> I tested and it doesn't look like we need to explicitly disable not-defined-by-default features...

Indeed, I see the `disabled` one as an example on how to explicitly disable other features that are enable by default.

> I slightly prefer the second option where the reference.conf lists all supported features

I bet new features will be forgotten...

The more I think about it, the more I think we should just rely on the `null` trick as laid out in the [HOCON doc](https://github.com/lightbend/config/blob/master/HOCON.md#duplicate-keys-and-object-merging).

Reply:
> Indeed, I see the disabled one as an example on how to explicitly disable other features that are enabled by default.

Got it, that makes sense.

> I bet new features will be forgotten...

Clearly :), but it's not an issue! We can add them when we realize we forgot some. I still think it's slightly better for users to have them all listed here, they clearly see what's disabled by default (experimental or obsolete).

> The more I think about it, the more I think we should just rely on the null trick

Meh. I prefer explicitly disabling stuff! It would be a problem if we had like 100 features, but given our rate of adding new feature bits we should be ok :)

Reply:
Fair enough, done in deca797b3.